src/main/resources/bundleconfiguration/default-config.yaml

```diff
@@ -0,0 +1,10 @@
+title: New bundle
+filename: bundle.pdf
+hasCoversheets: true
+hasTableOfContents: true
+documents:
+  - type: documentSet
+    property: /caseDocuments
+    filters:
+      - property: /name
```

Comment:
I know this is just an example but this should be `/documentName`

Reply:
OK